src/doc/rustc/src/platform-support/vxworks.md

```diff
@@ -20,6 +20,12 @@ Target triplets available:
 
 ## Requirements
 
+### OS version
+
+The minimum supported version is VxWorks 7 on x86/64 and ARM64.
```

Comment:
Why are the platforms specified - is something different for ppc and arm32?

Reply:
Its the same for all targets. 

Note: 
#128545

Reply:
Thanks for confirming. @devnexen could you drop the "on x86/64 and ARM64" unless there is a specific reason?

Reply:
@biabbas does this minimum version seem reasonable to you?

Also, could someone confirm when this version (VxWorks 7) was released? I am having some trouble figuring this out online what the oldest supported version will be.

Reply:
I think VxWorks 7 was released in 2016. Rust was enable for VxWorks in 2019(#61946 ). I've only worked on VxWorks 7.

Reply:
I think Versions older than VxWorks7 would require porting efforts, minimum supported version should be VxWorks7.

Reply:
I think this pr would cause merge conflicts with #128545.

Reply:
> I think Versions older than VxWorks7 would require porting efforts, minimum supported version should be VxWorks7.
But since we would have to go through the changes from VxWorks 6 to VxWorks 7 to confirm the minimum supported version. This seems less reasonable now.

Reply:
I think your PR should be merged first.

Reply:
@devnexen 
Do you have access to VxWorks 6 workbench?

Reply:
nope.

Reply:
> Thanks for confirming. @devnexen could you drop the "on x86/64 and ARM64" unless there is a specific reason?
After this, it seems ok for now.